feature-libs/product-configurator/rulebased/cpq/config/configurator-cpq.config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+
+export interface ProductConfiguratorCpqConfig {
+  cpqOrchestration?: {
```

Comment:
Is not enough to define a `cpqOverOcc` variable without additional structural complexity, namely nesting into  `cpqOrchestration`?

Reply:
I don't see an advantage of nexting